src/Catel.Core/Catel.Core.Shared/Logging/Listeners/FileLogListener.cs

```diff
@@ -129,6 +125,16 @@ public string FilePath
             }
         }
 
+        private void Initialize(bool initFilePath, Assembly assembly = null)
+        {
+           _assembly = assembly ?? AssemblyHelper.GetEntryAssembly() ?? Assembly.GetCallingAssembly();
+
+            if (initFilePath && string.IsNullOrWhiteSpace(_filePath))
+            {
+                _filePath = DetermineFilePath(AutoLogFileName);
```

Comment:
Just as comment, can you please make DetermineFilePath virtual (then users can easily override if they want to).